src/geo_types.rs

```diff
@@ -13,7 +15,7 @@
 /// assert_relative_eq!(result.x, 1450880.29f64, epsilon=1.0e-2);
 /// assert_relative_eq!(result.y, 1141263.01f64, epsilon=1.0e-2);
 /// ```
-impl<T: crate::proj::CoordinateType> crate::Coord<T> for geo_types::Coordinate<T> {
+impl<T: crate::proj::CoordinateType + Debug> crate::Coord<T> for geo_types::Coordinate<T> {
```

Comment:
Alternatively, you could consider making proj::CoorindateType itself implement Debug.

Reply:
Ha I was just hitting the "request review" button when this appeared.

Reply:
heh, I was investigating the CI failure and making recommendations when your commit appeared.

I guess good job GH?

Reply:
The advantage of making proj::CoordinateType implement debug is just in case you have some other proj code which is generic across proj::CoordinateType and you find yourself wanting to debug you won't have to later sprinkle in the Debug clarifications.

I think it's reasonable to require (as we have in geo) that whatever numeric types people ultimately plug in, must implement debug.

I cannot wait for the day when someone finally tells me that they are using geo w/ something besides f32/f64 - because we've definitely made the effort for them.

Reply:
@michaelkirk OK how about this (note the stricter bound on `project` et al now, though)